src/validation.c

```diff
@@ -3,23 +3,33 @@
 /*                                                        :::      ::::::::   */
 /*   validation.c                                       :+:      :+:    :+:   */
 /*                                                    +:+ +:+         +:+     */
-/*   By: pmarkaid <pmarkaid@student.hive.fi>        +#+  +:+       +#+        */
+/*   By: dbejar-s <dbejar-s@student.hive.fi>        +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/08/08 22:35:57 by pmarkaid          #+#    #+#             */
-/*   Updated: 2024/08/15 13:32:58 by pmarkaid         ###   ########.fr       */
+/*   Updated: 2024/08/21 01:54:26 by dbejar-s         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
+extern int	g_exit;
 
 static int	validate_access(char *file)
 {
+	char *msg;
+	
 	if (!access(file, F_OK))
 	{
 		if (!access(file, X_OK))
 		{
 			if (is_directory(file))
-				return (error_msg(file, 126));
+			{
+				msg = ft_strjoin("minishell: ", file, NULL);
```

Comment:
This I let like this,but alter they have to be handled with a proper error printer

Reply:
Yeah, I agree... it's just that I was not printing the right error message... so I hardcoded the whole thing. I can re-test and show the differences... actually managing with a proper error printer is better even if the message is not exactly the same (nobody would fail us for that).